web/__test__/Socials.test.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { describe, it, expect } from "vitest";
+import Socials from "../src/components/Socials";
+
+describe("Socials component", () => {
+	it("should render social media links", () => {
+		render(<Socials />);
+		const socialMediaLinks = [
+            {
+				name: "Instagram",
+				link: "https://www.instagram.com/au.indiansociety/?hl=en",
+			},
```

Comment:
Put all the social media links in the data file. This would create flaky tests if there is ever the change for any of the social media links causing unexpected behaviour